examples/query/react/suspense/src/Pokemon.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { pokemonApi } from './services/pokemon'
+import type { PokemonName } from './pokemon.data'
+
+const intervalOptions = [
+  { label: 'Off', value: 0 },
+  { label: '20s', value: 10000 },
+  { label: '1m', value: 60000 },
+]
+
+const getRandomIntervalValue = () =>
+  intervalOptions[Math.floor(Math.random() * intervalOptions.length)].value
+
+export const Pokemon = ({
+  name,
+  suspendOnRefetch = false,
+}: {
+  name: PokemonName
+  suspendOnRefetch?: boolean
+}) => {
+  const [pollingInterval, setPollingInterval] = React.useState(
+    getRandomIntervalValue()
+  )
+
+  const { data, isFetching, refetch } =
```

Comment:
is there a way to type `isFetching` as void or remove it from the return value if `suspendOnRefetch` is true? 

if `suspendOnRefetch` is `true`, `isFetching` is always going to be `false` when accessed in user code

Reply:
> is there a way to type `isFetching` as void or remove it from the return value if `suspendOnRefetch` is true?
> 
> if `suspendOnRefetch` is `true`, `isFetching` is always going to be `false` when accessed in user code

You can play with `ts` types in order to have `isFetching` be always true if `suspendOnRefetch` is set to `true` but,

no I don't think that I we should remove `isFetching` from the response.